### PR TITLE
Updates for v0.1.2 release

### PR DESCRIPTION
**Pull request by siggy**

### CODE REVIEW

**Comment by siggy** [comment]
@wmorgan please review the release notes. full changelog is at: https://github.com/runconduit/conduit/milestone/2?closed=1

File: CHANGES.md [hunk 1]
```diff
@@ -1,3 +1,20 @@
+## v0.1.2
+
+Conduit 0.1.2 is focused on usability and debuggability.
+
+* Conduit CLI
+  * New `conduit status` command reports on the health of your Conduit installation.
+  * New `conduit completion` command provides shell completion.
+* Dashboard
+  * Added scatterplot to deployments page.
```
Comment (rmars) [inline CHANGES.md]: Maybe also add
```
 * Added per-path metrics to the deployment detail pages
```

File: CHANGES.md [hunk 2]
```diff
@@ -1,3 +1,21 @@
+## v0.1.2
+
+Conduit 0.1.2 is focused on usability and debuggability.
+
+* Conduit CLI
+  * New `conduit check` command reports on the health of your Conduit installation.
+  * New `conduit completion` command provides shell completion.
+* Dashboard
+  * Added per-path metrics to the deployment detail pages.
+  * Added scatterplot to deployments page.
```
Comment (wmorgan) [inline CHANGES.md]: remove

File: CHANGES.md [hunk 3]
```diff
@@ -1,3 +1,21 @@
...
+  * Added animations to line graphs indicating server activity.
+  * More descriptive CSS variable names. (thanks @natemurthy!)
```
Comment (wmorgan) [inline CHANGES.md]: s/thanks/Thanks, and also for the two below

File: CHANGES.md [hunk 4]
```diff
@@ -1,3 +1,21 @@
...
+  * Surface all server errors.
+  * Removed iframe version check and JS errors.
```
Comment (wmorgan) [inline CHANGES.md]: I'd collapse these two into "A variety of other minor UI bugfixes and improvements"

File: CHANGES.md [hunk 5]
```diff
@@ -1,3 +1,21 @@
+## v0.1.2
+
+Conduit 0.1.2 is focused on usability and debuggability.
```
Comment (wmorgan) [inline CHANGES.md]: "Conduit 0.1.2 continues down the path of increasing usability and improving debugging and introspection of the service mesh itself."

File: CHANGES.md [hunk 6]
```diff
@@ -1,3 +1,21 @@
+## v0.1.2
```
Comment (wmorgan) [inline CHANGES.md]: Were there any data plane changes of note?

Comment (siggy) [reply CHANGES.md]: /cc @olix0r @carllerche @seanmonstar

Comment (carllerche) [reply CHANGES.md]: Nothing from me

Comment (siggy) [reply CHANGES.md]: No other changes.

**Pull request merged**